lib/from_openstudio/construction_set.rb

```diff
@@ -0,0 +1,141 @@
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require 'honeybee/construction_set'
+require 'from_openstudio/model_object'
+
+module Honeybee
+  class ConstructionSetAbridged < ModelObject
+
+    def self.from_construction_set(construction_set)
+        # create an empty hash
+        hash = {}
+        hash[:type] = 'ConstructionSetAbridged'
+        # set hash values from OpenStudio Object
+        hash[:identifier] = construction_set.nameString
+        hash[:wall_set] = {}
+        hash[:floor_set] = {}
+        hash[:aperture_set] = {}
+        hash[:door_set] = {}
+        hash[:roof_ceiling_set] = {}
+
+        # get interior surface constructions
+        unless construction_set.defaultInteriorSurfaceConstructions.empty?
+          int_surf_construction = construction_set.defaultInteriorSurfaceConstructions.get
+          # get interior wall construction
+          unless int_surf_construction.wallConstruction.empty?
+            int_wall_const = int_surf_construction.wallConstruction.get
+            hash[:wall_set][:interior_construction] = int_wall_const.nameString
+          end
+          # get interior floor construction
+          unless int_surf_construction.floorConstruction.empty?
+            int_floor_const = int_surf_construction.floorConstruction.get
+            hash[:floor_set][:interior_construction] = int_floor_const.nameString
+          end
+          # get interior roofceiling construction
+          unless int_surf_construction.roofCeilingConstruction.empty?
+            int_roof_const = int_surf_construction.roofCeilingConstruction.get
+            hash[:roof_ceiling_set][:interior_construction] = int_roof_const.nameString
+          end
+        end
+
+        # get interior subsurface constructions
+        unless construction_set.defaultInteriorSubSurfaceConstructions.empty?
+          int_subsurf_const = construction_set.defaultInteriorSubSurfaceConstructions.get
+          # TODO: check outdoor boundary condition for fixed window and operable window and assign to
```

Comment:
@chriswmackey the model schema notes that the interior_construction for aperture has an Outdoors boundary condition, False is_operable property, and a Wall face type for their parent face and the WindowConstruction has a Surface boundary condition. I think these two descriptions might need to be switched.

Reply:
Thanks, Tanushree. You are right that's a bug in the schema docs and I just pushed a fix:
https://github.com/ladybug-tools/honeybee-schema/pull/332
What you did here is correct.